### PR TITLE
App updated according to wireframe

### PR DESCRIPTION
**Pull request by developerPratik**
Updated CS1 branch for merging to dev branch

### CODE REVIEW

File: app/src/main/java/com/scanner/cardreader/CameraActivity.java [hunk 1]
```diff
@@ -42,68 +40,75 @@ protected void onCreate(Bundle savedInstanceState) {
...
 
 
 //Take a picture on button click
+
+        assert button!=null;
         button.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                camera.takePicture(shutterCallback, null,
+                camera.takePicture(null, null,
```
Comment (aviisekh) [inline app/src/main/java/com/scanner/cardreader/CameraActivity.java:96]: why did you remove shutterCallback

Comment (developerPratik) [inline app/src/main/java/com/scanner/cardreader/CameraActivity.java:96]: This feature was used to play a shutter click sound on image capture, now removed to reduce execution overhead.

Comment (aviisekh) [inline app/src/main/java/com/scanner/cardreader/CameraActivity.java:96]: Ehh. Thanks!!
